src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageSolutionDetailControlModel.cs

```diff
@@ -81,8 +81,13 @@ private void UpdateInstalledVersions()
         {
             var hash = new HashSet<NuGetVersion>();
 
+            IsRequestedVisible = false;
             foreach (var project in _projects)
             {
+                if (IsRequestedVisible.Equals(false) && project.NuGetProject.ProjectStyle.Equals(NuGet.ProjectModel.ProjectStyle.PackageReference))
```

Comment:
In my earlier suggestion, I was thinking to simply set IsRequestedVisible inside of the `if` statement you already have here: https://github.com/NuGet/NuGet.Client/pull/3555/commits/bfd4f346c9c377deeab4ac8b2098aa6aea538855#diff-17f11a93b3135e66bd31d0d723de815aR99

Reply:
The link is not taking to a specific line of code, but I think you are referring to the if below where I check the project type, that is inside a try so it can fail

Reply:
I'm going to add an image to be more explicit, but you can have the requested column even though you don't have package with a requested version or installed, this column is going to show if there is a PR project so we don't confuse users on when this column is being show. @nkolev92